### DISCUSSION TITLE
FP V 1.10.6 Tutorial Filter map opening error

### DISCUSSION THREAD

**Discussion by dsmithcsepINCOSE56** [freeplane/freeplane#985] [Help]
Trying to get a handle on using filters for searching node attributes. (Windows)

In the tutorial mind map is reference to a filter tutorial file "freeplane Tutorial.mmfilter" file.  The file exists in C: > Program Files > Freeplane > doc  However it can't be opened via File >  Open Mind Map ... menu item because it does not have a *.mm file extension.  

I tried copying the file to my project working directory where my other *.mm files are stored and get the error "Error while parsing file:   ......  I tried renaming the file name to have a *.mm file type and that doesn't work either.

Also, the link "Select filter and Quickfilter does not open - suspect that it only refers to that item in the filter tutorial mm.

**To Reproduce**
Follow the described process above.

**Expected behavior**
To be able to use the file to learn how to use filters.  
**Mind map files**
see the installed director as described above.

**Screenshots**
See attached image

**Desktop (please complete the following information):**
 - Windows 10
 - 
 - Java Version
Java 8 Update 351 64 bit
![2023-02-04 Can not load filter tutorial file](https://user-images.githubusercontent.com/123216787/216801467-91f8f0db-ef59-477f-b4f7-0340f16f4f0c.png)
![2023-02-04_Filter Tutorial Instructions](https://user-images.githubusercontent.com/123216787/216801468-b1103469-daf0-4389-924e-69bf0dcb6268.jpg)

**Additional context**

**Comment by dpolivaev** [comment]
Use the "load" button in the filter composer dialog, Filter->Compose filter
